README.md

```diff
@@ -15,10 +15,70 @@
 ```bash
 $ npm install @adobe/helix-querybuilder
 ```
+## Background
+
+The [AEM QueryBuilder](https://docs.adobe.com/content/help/en/experience-manager-65/developing/platform/query-builder/querybuilder-api.html) is a Java and REST API for executing server-side queries using a custom Query Builder Language (QBL). QBL was designed to be:
+
+1. implementation agnostic
+2. HTML-form friendly (you should not need JavaScript to build a query)
+3. simple (no joins or projections)
+
+## QBL Language and Notation
+
+### As URL Query String
+
+The most common way of expressing queries is as a query string appended to the URL of the resource that is able to execute queries. In the context of Project Helix, this would be the [Helix Data Embed Action](https://github.com/adobe/helix-data-embed#filtering-results).
+
+An example query might look like this:
+
+```
+https://adobeioruntime.net/api/v1/web/helix/helix-services/data-embed@v1/https://blogs.adobe.com/psirt/?feed=atom&hlx_property=author&hlx_property.value=svishnoi
```

Comment:
I don't like to advertise the non-escaped variant...although it makes the example more readable.
also, mentioning `data-embed` here is only confusing. better just use:

```
https://blogs.adobe.com/psirt/?feed=atom&hlx_property=author&hlx_property.value=svishnoi
```

Reply:
I picked the non-escaped variant because it is easier to read. Just using the feed URL won't work because the feed itself doesn't do any filtering.